docs/website/docs/user-guides/advanced/connecting-a-service.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Connecting to a Service
+sidebar_position: 1
+---
+
+Building on top of the [Go quickstart guide](../quickstart/go.md), this guide will extend the application to connect it to a deployable MongoDB service.
+
+## Prerequisites
+1. [Install the Service Binding Operator via Operator Hub](https://operatorhub.io/operator/service-binding-operator). Read [here](../../command-reference/add-binding.md#description) to know why this is required.
+
+2. [Install Percona Server Mongodb Operator via Operator Hub](https://operatorhub.io/operator/percona-server-mongodb-operator).
+:::note
+The operator will be installed in a new namespace called "my-percona-server-mongodb-operator" and will be usable from that namespace only.
```

Comment:
Won't this mess up deployment with the service if it's only usable in that namespace?

Maybe we should add instructions to deploy / use this Go application in the same namespace as the operator?

Reply:
I just realized that we don't need to. A service deployed in any namespace is still accessible to the Go application deployed in a different namespace.